unicycler/unicycler.py

```diff
@@ -222,6 +223,9 @@ def main():
     graph.save_to_gfa(final_assembly_gfa)
     graph.save_to_fasta(final_assembly_fasta, min_length=args.min_fasta_length)
 
+    # Align reads to assembly.fasta and report depth
+    get_read_depth(final_assembly_fasta, args)
```

Comment:
Due to the reliance on assembly.fasta, read depth is calculated at the very end of the pipeline. Thus, the log header "Calculating read depth" and the depth information comes after the "Assembly complete" header. Maybe this behavior should be modified.